packages/test-utils/src/wrapper.js

```diff
@@ -711,7 +711,15 @@ export default class Wrapper implements BaseWrapper {
 
       // $FlowIgnore : Problem with possibly null this.vm
       this.vm.$forceUpdate()
-      return nextTick()
+      return new Promise((resolve, reject) => {
+        nextTick().then(() => {
+          const isUpdated = Object.keys(data).some(key => {
+            // $FlowIgnore : Problem with possibly null this.vm
+            return this.vm[key] === data[key]
```

Comment:
not sure if we need to compare by value here, but figured this is handled above? Might also be a if the same object property is set, but another is not.

Reply:
This is comparing by value right?

Reply:
@lmiller1990 this would be a comparison by reference if we are considering objects/array. But these should be assigned by reference [here](https://github.com/vuejs/vue-test-utils/blob/dev/packages/test-utils/src/wrapper.js#L709) which makes me think this should be enough to check the reference, unless something is getting mutated under the hood? Might be good to do a value check here to be safe